### PR TITLE
fix(admin-ui): Patch admin path issue

### LINKED ISSUE #5151: Medusa Admin fails to fetch language files
Hi!

I am getting the following errors in my browser when i18next is trying to fetch language files:

Screenshot 2023-09-19 at 21 25 26
Screenshot 2023-09-19 at 21 25 36

As you can see there is a double slash after the backend URL which I think has something to do with the fact that in https://github.com/medusajs/medusa/blob/889154c440e3a95042979d78446eb5f3f5585020/packages/admin-ui/ui/src/i18n/index.ts#L22 there is a trailing slash when the ADMIN_PATH has a default value of '/'

### PR DESCRIPTION
**Pull request by olivermrbl**
Closes #5151

This was an issue two places:
- Path for loading translations
- Building the invite link to send to invited users

### CODE REVIEW

File: packages/admin-ui/ui/src/utils/get-admin-path.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export function getFullAdminPath() {
+  if (window) {
+    const origin = window.location.origin // returns base url with trailing slash
+
+    const adminPath = process.env.ADMIN_PATH ? `${process.env.ADMIN_PATH}/` : ""
+
+    let fullPath = `${origin}${adminPath}`
+
+    // safeguard against double slashes, which happens in case path is set to `/`
+    if (fullPath.endsWith("//")) {
```
Comment (adrien2p) [inline packages/admin-ui/ui/src/utils/get-admin-path.ts:14]: Suggestion(non blocking): fullPath.replaceAll(´//´, ´/´)

To also prevent unwanted double slash anywhere

File: packages/admin-ui/ui/src/utils/get-admin-path.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/**
+ * returns the full admin path using the window object
+ * @returns {string} - admin path
+ */
+export function getFullAdminPath() {
```
Comment (adrien2p) [inline packages/admin-ui/ui/src/utils/get-admin-path.ts:5]: Suggestion: getAdminPath? Don't seem that we ever get non full path no?

**Review by adrien2p — APPROVED** [review]
LGTM

File: packages/admin-ui/ui/src/utils/get-admin-path.ts (hunk 2 above)
Comment (olivermrbl) [reply packages/admin-ui/ui/src/utils/get-admin-path.ts:5]: Admin path in the context of our plugin is whatever is provided in the `path` config in `medusa-config.js`. With window.location.origin, this becomes a "full" admin root path.

File: packages/admin-ui/ui/src/utils/get-admin-path.ts (hunk 1 above)
Comment (olivermrbl) [reply packages/admin-ui/ui/src/utils/get-admin-path.ts:14]: I don't think we necessarily want to replace all. We just need to make sure, that whatever is appended to the full admin path is appended correctly i.e. with no double slashes.

File: packages/admin-ui/ui/src/utils/get-admin-path.ts (hunk 2 above)
Comment (olivermrbl) [reply packages/admin-ui/ui/src/utils/get-admin-path.ts:5]: Do you think we should change it to `getRootAdminPath`?

Comment (adrien2p) [reply packages/admin-ui/ui/src/utils/get-admin-path.ts:5]: I think we cant keep it like it 👍

**Pull request merged**